google-cloud-video_intelligence/README.md

```diff
@@ -32,7 +32,7 @@ features_element = :LABEL_DETECTION
 features = [features_element]
 
 # Register a callback during the method call.
-operation = video_intelligence_client.annotate_video(input_uri: input_uri, features: features) do |op|
+operation = video_intelligence_client.annotate_video(features, input_uri: input_uri) do |op|
```

Comment:
@dazuma These API breaking changes are expected, yes?

Reply:
@blowmage It looks like it was intentional. (The parameter should have been listed as required since the backend would error out if it's not present, so they were trying to "correct" it.) They weren't expecting it to affect client libraries, and in fact it looks like Ruby is the only one affected because of how the argument flattening is implemented. (One more reason I'm glad we got rid of it in the microgenerator.)

Since we're preparing a 2.0 release anyway, I think we should just let this through and mark it as breaking.